mlir-tensorrt/compiler/include/mlir-tensorrt/Compiler/Options.h

```diff
@@ -34,6 +34,9 @@ namespace mlirtrt::compiler {
 /// DebugOptions are options that are common to different compiler API
 /// interfaces.
 struct DebugOptions {
+  /// Dump textual pipeline passes
+  bool dumpTextualPipeline = false;
```

Comment:
@christopherbate I will move such changes to NFC PR's once we finalize the implementation.